packages/angular/src/generators/stories/lib/tree-utilities.ts

```diff
@@ -0,0 +1,21 @@
+import type { Tree } from '@nrwl/devkit';
+import { joinPathFragments } from '@nrwl/devkit';
+
+export function getAllFilesRecursivelyFromDir(
```

Comment:
I think we might be able to leverage this function instead of having a custom one, what do you think?

https://github.com/nrwl/nx/blob/master/packages/devkit/src/generators/visit-not-ignored-files.ts#L8

Reply:
Yeah, I forgot about this one, though I would prefer to use something else that allows me to break the loop when a condition is met. With the `visitNotIgnoreFiles`, I have to visit all files and execute the visitor function on them, even if I know that is not needed because a condition was previously met. One of the use cases in this generator can break before processing all files.

Reply:
I see, sounds good 👍